include/nuttx/fs/fs.h

```diff
@@ -425,6 +425,55 @@ struct inode
 
 #define FSNODE_SIZE(n) (sizeof(struct inode) + (n))
 
+/* Definitions for custom stream operations with fopencookie. The
+ * implementation is as defined in Standard C library (libc). The only
+ * difference is that we use off_t instead of off64_t. This means
+ * off_t is int64_t if CONFIG_FS_LARGEFILE is defined and int32_t if not.
+ */
+
+typedef ssize_t cookie_read_function_t(void *cookie, char *buf, size_t size);
+typedef ssize_t cookie_write_function_t(void *cookie, const char *buf,
+                                        size_t size);
+typedef off_t cookie_seek_function_t(void *cookie, off_t *offset,
```

Comment:
```suggestion
typedef CODE int cookie_seek_function_t(FAR void *cookie, FAR off_t *offset,
```
add CODE to ALL function typdef

Reply:
done